File: frontend/ultima-ng-19.0.0/src/app/interface/creditRequest.ts
import { RequisitoRequest } from './requisitoRequest';

export class CreditRequest {
    codAgencia?: string;
    tipCredito?: number;
    codCliente?: string;
    monCredito?: number;
    cantCuotas?: number;
    codUsuario?: string;
    codOrigen?: string;
    codPlanInversion?: string;
    codPlazo?: string;
    codActividad?: string;
    codSubactiv?: string;
    codSubsubactividad?: string;
    codTasaInt?: string;
    codTasaMora?: string;
    tipModalidadCobro?: string;
    tipInteres?: string;
    tipCalendario?: number;
    tipCuota?: string;
    plazoCredito?: number;
    tipTasa?: string;
    tasaInteres?: number;
    tasaMora?: number;
    indLinea?: string;
    observaciones?: string;
    codEjecutivo?: string;
    requisitos?: RequisitoRequest[];
}

File: micro-digiservices/clients/src/main/java/io/digiservices/clients/domain/CreditRequest.java
package io.digiservices.clients.domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.*;

import java.math.BigDecimal;
import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class CreditRequest {
    private String codAgencia;
    private Integer tipCredito;
    private String codCliente;
    private BigDecimal monCredito;
    private Integer cantCuotas;
    private String codUsuario;
    private String codOrigen;
    private String codPlanInversion;
    private String codPlazo;
    private String codActividad;
    private String codSubactiv;
    private String codSubsubactividad;
    private String codTasaInt;
    private String codTasaMora;
    private String tipModalidadCobro;
    private String tipInteres;
    private Integer tipCalendario;
    private String tipCuota;
    private Integer plazoCredito;
    private String tipTasa;
    private BigDecimal tasaInteres;
    private BigDecimal tasaMora;
    private String indLinea;
    private String observaciones;
    private String codEjecutivo;
    private List<RequisitoRequest> requisitos;
}

